S/SCALE_MAMBA/build_tarballs.jl

```diff
@@ -35,16 +36,13 @@ fi
 make -j${nproc} CC="${CXX}" -C src
 
 # Install executables into ${prefix}
-mkdir -p ${prefix}/bin
-cp Player.x${exe} ${prefix}/bin/
-cp Setup.x${exe} ${prefix}/bin/
+install -Dvm 755 "Player.x${exe}" "${bindir}/Player.x${exe}"
```

Comment:
Did you mean exeext?

Reply:
I have no idea. This was you:
https://github.com/JuliaPackaging/Yggdrasil/pull/6532/commits/c01efa6b231b0f41f7c36e764fa3614ee65b8366

Reply:
The environment variable I mean, there's no exe, only exeext (and I didn't touch it there): https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables. It doesn't make much difference here since exeext is the empty string for Linux, but using a non-existing variable doesn't do any good